models/zeroex/arbitrum/zeroex_arbitrum_native_fills.sql

```diff
@@ -228,7 +228,7 @@ WITH
                 taker_token_filled_amount_raw as taker_token_amount_raw,
                 maker_symbol,
                 token_pair,
-                CAST(ARRAY() as array<bigint>) as trace_address,
+                '1' as trace_address,
```

Comment:
we had been defaulting this value to empty string, `""`, but we will be deprecating this field in the `dex.trades` redesign in the near future as it's not used across any projects added so far. with that in mind, i think it's okay to leave this as-is and let it go